docs/deploy.md

```diff
@@ -0,0 +1,131 @@
+# Procedure for releasing new AMY version
+
+1. Close milestone using [GitHub UI](https://github.com/swcarpentry/amy/milestones).
+
+1. Follow procedure for deployment on production.
+
+1. Write down release notes and publish it on [Piotr's blog](http://piotr.banaszkiewicz.org/). Release notes should have two sections: new features and bugfixes, each one enumerating changes and mentioning their authors. Base your work on the list of all closed pull request for given milestone, available on GitHub.
+
+# Procedure for deployment on production
+
+Execute the following commands on your local machine, not production.
+
+1.  Release AMY v1.X.Y and version bump on master
+
+    1.  Move to AMY root directory (the one with `manage.py` file) using `cd` command.
+    
+    1.  Make sure you have configured repositories:
+    
+        - `origin` for your repository on GitHub
+        - `upstream` for [`swcarpentry/amy` repo on GitHub](https://github.com/swcarpentry/amy)
+        
+        For example, this is the correct configuration for `chrismedrela`:
+        
+            $ git remote -v
+            origin	git@github.com:chrismedrela/amy.git (fetch)
+            origin	git@github.com:chrismedrela/amy.git (push)
+            upstream	git@github.com:swcarpentry/amy.git (fetch)
+            upstream	git@github.com:swcarpentry/amy.git (push)
+    
+    1.  Make sure your local `develop` and `master` branches are up to date:
+    
+            $ git checkout develop
+            $ git pull upstream develop
+            $ git push origin develop
+            
+            $ git checkout master
+            $ git pull upstream master
+            $ git push origin master
+        
+        Pushes to your `origin` repository are optional.
+        
+    1.  Update `master` branch:
+        
+            $ git checkout master
+            $ git merge --no-ff develop  # sometimes there are conflicts in files with release version, but most often we need to edit them and commit
+            
+    1.  Bump version on `master`:
+    
+            $ vim bower.json  # change 'version' to 1.X.Y
+            $ vim workshop/__init__.py  # change version to 1.X.Y
+            $ git add workshops/__init__.py bower.json
+            $ git commit -m "Bumping version to v1.X.Y"
+    
+    1.  Just to be safe, run tests.
+    
+            $ make test
+    
+    1.  Tag a release. Tag message should be `AMY release v1.X.Y`. 
+        
+            $ git tag -a v1.7.1 -s
```

Comment:
In [line 52](https://github.com/swcarpentry/amy/pull/1077/files#diff-ef860daacd753531645fa756fbe5b836R52) you have `v1.X.Y` but here you use explicit values for X and Y: `v1.7.1`. Maybe it's a good idea to keep X and Y everywhere or the same example `v1.7.1` everywhere?